src/nidmm/metadata/__init__.py

```diff
@@ -10,12 +10,7 @@
 
 # Update generated functions data with hand maintained data
 config['modules'] = sys.modules
-helper.add_all_function_metadata(functions, config)
-helper.add_all_attribute_metadata(attributes, config)
+helper.add_all_metadata(functions, attributes, enums, config)
```

Comment:
I like how this reduces duplication. I still wish we could single source these __init__.py